### DISCUSSION TITLE
Has anyone gotten messages to work

### DISCUSSION THREAD

**Discussion by Sigma-dev** [Thinkofname/steamworks-rs#206] [Q&A] [answered]
I've been trying to get messages to work on my own application for a while but I can't seem to.

So I'm just looking for a sanity check that the SteamNetworking Messages api is still working message with steamworks-rs.

An example would also be vastly appreciated.

**Comment by Noxime** [comment, marked as answer]
Hello, there was a bug with the session acceptance logic. It has been fixed in #207 . When accepting the new peer session, the connection was immediately closed. I've also added a new example to showcase the API

**Reply by Sigma-dev** [reply]
Thanks mate, I've been struggling on this for weeks, so i'm happy it's over.
